packages/config/config/devices/0x0312/mp22z.json

```diff
@@ -8,104 +8,79 @@
-		},
-		{
-			"#": "16",
-			"$import": "~/templates/master_template.json#dimming_speed_1-99_seconds"
-		},
-		{
-			"#": "20",
-			"$import": "~/0x0312/templates/minoston_template.json#association_reports"
-		},
-		{
-			"#": "22",
-			"$import": "~/0x0312/templates/minoston_template.json#night_mode_brightness"
+			"#": "2",
+			"label": "Auto-Off Timer",
+			"valueSize": 2,
+			"unit": "minutes",
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
 		},
 		{
-			"#": "23",
-			"$import": "~/0x0312/templates/minoston_template.json#led_indicator_color"
+			"#": "4",
+			"label": "Auto-On Timer",
+			"valueSize": 2,
+			"unit": "minutes",
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
 		},
 		{
-			"#": "26",
-			"$import": "~/0x0312/templates/minoston_template.json#led_indicator_brightness"
+			"#": "6",
+			"$import": "~/templates/master_template.json#state_after_power_failure_off_on_prev",
+			"defaultValue": 0
 		}
 	],
 	"metadata": {
-		"inclusion": "1. Refer to your primary controller instructions to process the inclusion / exclusion setup procedure.\n2. When prompted by your primary controller, click the button three times in one second",
-		"exclusion": "1. Refer to your primary controller instructions to process the inclusion / exclusion setup procedure.\n2. When prompted by your primary controller, click the button three times in one second",
-		"reset": "Press click Z-Wave button 3 times quickly, and hold for at least 10 seconds at the third time to restore the device to the factory\n(Node:Please use this procedure only when the network primary controller is missing or otherwise inoperable.)",
-		"manual": "https://products.z-wavealliance.org/ProductManual/File?folder=&filename=product_documents/4254/MP22ZD-manual-20210728.pdf"
+		"inclusion": "ADD the MP22Z from the existing Z-Wave home control network with your primary controller.\n ---Refer to your primary controller instructions to process the inclusion / exclusion setup procedure.\n ---When prompted by your primary controller, click the Up or Down button three times in one second.\nInclude MP22Zto/from a Z-Wave Gateway with supporting Security.The MP22Zcan support the Primary Controller that implemented the security S2",
+		"exclusion": "Remove the MP22Zfrom the existing Z-Wave home control network with your primary controller.\n ---Refer to your primary controller instructions to process the inclusion / exclusion setup procedure.\n ---When prompted by your primary controller, click the Up or Down button three times in one second.\nInclude MP22Zto/from a Z-Wave Gateway with supporting Security.The MP22Z can support the Primary Controller that implemented the security S2",
```

Comment:
The inclusion/exclusion instructions are formatted worse than before. Please try to use the old formatting with 1./2./...